extension/functions/src/postgres.rs

```diff
@@ -14,3 +14,143 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
+use arrow::array::*;
+
+use arrow::array::{ArrayRef, TimestampNanosecondArray};
+use arrow::compute::cast;
+use arrow::datatypes::IntervalDayTimeType;
+use arrow::datatypes::IntervalUnit;
+use arrow::datatypes::{DataType, TimeUnit};
+use datafusion::error::Result;
+
+use chrono::NaiveDateTime;
+use datafusion::logical_expr::Volatility;
+
+use datafusion_common::DataFusionError;
+use datafusion_expr::{
+    ReturnTypeFunction, ScalarFunctionDef, ScalarFunctionPackage, Signature,
+};
+use std::sync::Arc;
+
+#[derive(Debug)]
+pub struct AgeFunction;
```

Comment:
Could you add a link to the function in Postgres doc? That makes it easier to cross reference the source of truth. 
e.g.
```suggestion
/// Postgres `age` function ([link to doc])
pub struct AgeFunction;
```

Reply:
Great point! cc @xiaoyumoyu

Reply:
ok！Should I submit it again right away after making the additions？